common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -359,6 +367,14 @@ private void launchCompanyPortal() {
         returnResult(RawAuthorizationResult.ResultCode.MDM_FLOW);
     }
 
+    private void processAmazonAppUri(@NonNull final String url) {
```

Comment:
Do we need to explicitly stop/close the webview after this, or that is something that is already happening.

Reply:
No. The webview needs to keep running. The user returns back to the flow after completing action in AEA app.

Reply:
I see, so the web view is showing a page that indirectly waits for user performing an action in AEA app.

Reply:
Yes, correct.